storage/imx6ull-flash/flashdrv.c

```diff
@@ -1014,7 +1014,7 @@ static void setup_flash_info(void)
 
 	if (flash_id->manufacturerid == 0x98 && flash_id->deviceid == 0xd3) {
 		flashdrv_common.info.name = "Kioxia TH58NV 16Gbit NAND";
-		flashdrv_common.info.size = 4096ull * 64 * 8192;
+		flashdrv_common.info.size = 4096ull * 64 * 4096;
```

Comment:
please write comment WHY this change is needed - we don't want it accidentally "fixed" later

Reply:
Done.